tasks/microbenchmark/Cargo.toml

```
@@ -1,5 +1,5 @@
 [package]
-name                  = "pacquet_benchmark"
+name                  = "pacquet_microbenchmark"
```

To be in correlation with `pacquet_integrated_benchmark`

```suggestion
name                  = "pacquet_micro_benchmark"
```

---

Done.